src/Async.js

```diff
@@ -178,8 +183,11 @@ export default class Async extends Component {
 			onInputChange(transformedInputValue);
 		}
 
+		let oldInputValue = this.state.inputValue;
```

Comment:
setting state is async. This with the check two lines below will lead to bugs where the loadOptions is not called when it should have been.